docs/continuous.rst

```diff
@@ -56,6 +57,9 @@ The :py:mod:`stochastic.continuous` module provides classes for generating
 
 .. autoclass:: stochastic.continuous.PoissonProcess
     :members: rate, sample
+    
+.. autoclass:: stochastic.continuous.MixedPoissonProcess
+    :members: rate_func,rate_args,rate_kwargs,gen_rate
```

Comment:
Should only expose the attributes/methods that should be public, here we really only want rate, rate_func, rate_args, rate_kwargs, sample.